build/common/installer/scripts/tomlparser.rb

```diff
@@ -215,7 +215,7 @@ def populateSettingValuesFromConfigMap(parsedConfig)
         @logEnableKubernetesMetadata = parsedConfig[:log_collection_settings][:metadata_collection][:enabled]
         if !parsedConfig[:log_collection_settings][:metadata_collection][:include_fields].nil?
           puts "config::Using config map setting for kubernetes metadata include fields"
-          @logKubernetesMetadataiIncludeFields = parsedConfig[:log_collection_settings][:metadata_collection][:include_fields]
+          @logKubernetesMetadataiIncludeFields = parsedConfig[:log_collection_settings][:metadata_collection][:include_fields].join(",")
```

Comment:
should we check whether the input is a valid array before doing a join?

Reply:
added